index.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable linebreak-style */
+import { DateTime } from './modules/luxon.js';
+
+import BookShelf from './modules/books.js';
+
+const dt = DateTime.now();
+
+document.querySelector('.date').innerHTML = `${dt.day} ${dt.month}, ${dt.year} ${dt.hour}:${dt.minute}:${dt.second}`;
+
+const menuItems = document.querySelectorAll('li');
+const sections = document.querySelectorAll('section');
+function reset() {
+  sections.forEach((section) => {
+    section.style.display = 'none';
+  });
+}
```

Comment:
- [ ] In keeping with the ES6 syntax requirement, kindly refactor this function to an arrow function.

Reply:
> * [ ]  In keeping with the ES6 syntax requirement, kindly refactor this function to an arrow function.

@TedLivist Thank you for pointing out reset function which was still to convert in arrow function. I have done and submitted again.

Reply:
well resolved! ✔️